docs/modules/panel_configuration/pages/index.adoc

```diff
@@ -19,22 +19,116 @@ This section describes the following types of panel configuration:
 [[custom-panel]]
 == Default custom panels
 
-{page-component-title} comes with several pre-configured, custom panels.
-They are described in this section.
+{page-component-title} comes with several pre-configured custom panels:
+
+* *Alarm Histogram:* A bar chart that shows the distribution of alarms by severity level.
++
+image::alarm-histogram.png["An Alarm Histogram panel displaying a horizontal bar graph. The bar graph shows the distribution of critical and major alarms.", 850]
 
-Alarm Histogram:: Bar chart that shows the distribution of alarms by severity level.
+* *Alarms Table:* A table that displays alarms.
```

Comment:
I would mention that the alarms table also can display any information associated to the alarm, like status, location, node assciated info, assets associated info (whatever is allowed in the panel options) that the user may find relevant. And also their status can also be interacted from the panel (like aknowledge the alarm, clear the alarm, add memos, etc)